CONTRIBUTING.md

```diff
@@ -36,3 +36,22 @@ The main files are:
 9. `x64/` is the directory with all the machine code generation logic. It will be available in early July. Obviously this is the most complex part of the compiler. It defines a set of functions that translates assembly instructions to machine code, it builds complicated binaries from scratch byte by byte. It manually builds all headers, segments, sections, symtable, relocations, etc. Right now it only has basic support of the x64 platform/Mach-O format, and it can only generate `.o` files, which then have to be linked with `lld`.
 
 The rest of the directories are vlib modules: `builtin/` (strings, arrays, maps), `time/`, `os/`, etc. Their documentation is pretty clear.
+
+## Example Workflow for Contributing
+##### (provided by [@spytheman](https://github.com/spytheman))
+
+1. Clone https://github.com/vlang/v in a folder, say nv (`git clone https://github.com/vlang/v nv`)
+1. `cd nv`
```

Comment:
~The list points should be numbered, since in some steps I referred to the previous ones by number.~
It seems that when rendered, the 1 in the markdown gets renumbered by github, so that may not be needed.